python/sglang/srt/layers/layernorm.py

```diff
@@ -78,6 +78,26 @@ def forward_native(
         else:
             return x, residual
 
+    def forward_hpu(
```

Comment:
Hi @pi314ever would you mind rebasing on #12 . it uses a release version of vllm-fork. https://github.com/HabanaAI/vllm-fork/releases/tag/v0.6.6.post1%2BGaudi-1.20.0

I think we can upgrade when vllm-fork release another tag.